cmake/toolchain.cmake

```diff
@@ -3,7 +3,7 @@
 
 option(MBEDIDE "Use Arm compiler from Mbed Studio" OFF)
 if(MBEDIDE)
-    set(MBED_STUDIO_ARM_COMPILER "--ide=mbed")
+    set_property(GLOBAL PROPERTY MBED_STUDIO_ARM_COMPILER "--ide=mbed")
```

Comment:
should this be global or target (mbed-os) specific?

Reply:
In my opinion, global as it applies globally to anything built with ARM Compiler 6 and not just to a build target.
However, if you feel differently about it and want it changed tell me your reasons and we can change it.